spec/validators/created_in_past_validator_spec.rb

```diff
@@ -29,13 +29,41 @@
       end
     end
 
-    context 'with a date after the current year' do
+    describe 'validate date is in the past' do
       let(:attribute) { :created_edtf }
-      let(:year) { Time.zone.today.year + 1 }
-      let(:value) { EDTF.parse("#{year}-01-01") }
+      let(:today) { Time.zone.today }
 
-      it 'has errors' do
-        expect(record.errors.full_messages).to eq ['Created edtf must be in the past']
+      context 'with a date after the current year' do
+        let(:year) { today.year + 1 }
+        let(:value) { EDTF.parse(year.to_s) }
+
+        it 'has errors' do
+          expect(record.errors.full_messages).to eq ['Created edtf must be in the past']
```

Comment:
Is this a useful error message?

Reply:
For a developer, sure. For an end-user, not so much.

Reply:
The end user will never look at `full_messages`, so no problem.